frontend/src/pages/MessagingQueues/MessagingQueues.tsx

```diff
@@ -192,7 +192,7 @@ function MessagingQueues(): JSX.Element {
 							</Button>
 						</div>
 					</div>
-					<div className="overview-info-card">
+					<div className="overview-info-card middle-card">
```

Comment:
`middle-card` is being used to add additional styling, you can use nth child and access element directly

Reply:
done